modeshape-jcr/src/main/java/org/modeshape/jcr/value/binary/S3BinaryStore.java

```diff
@@ -73,9 +77,9 @@
     protected static final String USER_MIME_TYPE_KEY = "user-mime-type";
 
     /*
-     * Key for storing boolean which describes if object is unused
+     * Tag key for storing boolean which describes if object is unused
      */
-    protected static final String UNUSED_KEY = "unused";
+    protected static final String UNUSED_TAG_KEY = "modeshape.unused";
```

Comment:
Wouldn't changing this value break backward compatibility in some way for existing users of the S3 store? Is there a straightforward migration mechanism?

Reply:
After further understanding the differences between the current code and your work, existing backend storage would break, but not in the way I was thinking. Currently unused values are marked metadata, while you have tried here to mark them using tags, presumably because tags can be filtered by lifecycle rules. So it's not that you've changed the name of the tag, as I originally thought, but that you've changed the marking mechanism. I think we could handle that with an upgrade operation.